web/src/components/MapView.tsx

```diff
@@ -91,41 +91,33 @@ const MapView = forwardRef((props: MapViewProps, ref: React.Ref<MapRef>): ReactE
   const [cursor, setCursor] = useState<MapCursorType>('auto')
   const theme = useTheme()
   const navigate = useNavigate()
+  const location = useLocation()
 
   const { viewportSmall } = useWindowDimensions()
 
-  const onDeselectFeature = useCallback(
```

Comment:
this seem not to be needed anymore, since the MapControl do not affect a deselection maybe due to lib version update, so i added a regular `onDeselect`  in `onSelectFeature`